components/brave_rewards/resources/ui/components/adsBox.tsx

```diff
@@ -113,7 +125,25 @@ class AdsBox extends React.Component<Props, State> {
         onToggle={this.onAdsSettingChange.bind(this, 'adsEnabled', '')}
         settingsOpened={this.state.settings}
         onSettingsClick={this.onSettingsToggle}
-      />
+      >
+        <List title={getLocale('adsCurrentEarnings')}>
+          <Tokens
+            value={adsEstimatedEarnings.toString()}
+            converted={utils.convertBalance(adsEstimatedEarnings.toString(), walletInfo.rates)}
+          />
+        </List>
+        <List title={getLocale('adsPaymentDate')}>
+          <NextContribution>
+            {'Monthly, 5th'}
```

Comment:
@jsecretan is this date final?

Reply:
Yep, this is the date we are going with.  Could technically change in the future, but this would be the only place to change it as the grant polling should take care of the rest.